app/schemas/uiSchema.js

```diff
@@ -1,22 +1,58 @@
 const uiSchema = {
-  "nonISPCollaborationPlan": {
-    "ui:widget": "textarea"
+  "Zone Feedback": {
+    "ui:widget": "textarea",
+    "ui:options": {
+      "cols": 100
```

Comment:
is the cols section working? What about rows? If not let's get a ticket open

Reply:
cols and rows not working, will open ticket